index.js

```diff
@@ -41,13 +42,136 @@ const ABP_DETECTION_PX = "https://media.ethicalads.io/abp/px.gif";
 // here for future support.
 const SUPPORTS_MULTIPLE_PLACEMENTS = false;
 
+// Keywords and topics
+//
+// This allows us to categorize pages simply and have better content targeting.
+// Additional categorization can be done on the server side for pages
+// that request ads commonly but this quick and easy categorization
+// works decently well most of the time.
+const KEYWORDS = {
```

Comment:
Could use a mapping -- I assume the key is the page copy and the value is the server-side keyword?

Reply:
Correct.